shared/common-adapters/standard-screen.stories.tsx

```diff
@@ -31,14 +31,14 @@ const load = () => {
     ))
     .add('Back Button', () => (
       <Wrapper>
-        <StandardScreen {...props} onClose={null} onBack={action('onBack')} />
+        <StandardScreen {...props} onClose={undefined} onBack={action('onBack')} />
```

Comment:
is there a difference between including `onClose` as `undefined` and not including it?

Reply:
no it could be removed

Reply:
actually i take this back. normally it would be but the `{...props}` means it could be set, and this story intended to unset it so it has to be explicitly set. aka `props.conClose={Sb.Action('onClose')}` needs to be overwritten